packages/file-asset-apis/src/s3/createS3Client.ts

```diff
@@ -2,17 +2,17 @@ import fs from 'fs-extra';
 import { Agent, AgentOptions as HttpsAgentOptions } from 'https';
 import { S3Client as BaseClient } from '@aws-sdk/client-s3';
 import { NodeHttpHandler, NodeHttpHandlerOptions } from '@smithy/node-http-handler';
-import type { S3ClientConfig as baseConfig } from '@aws-sdk/client-s3';
+import type { S3ClientConfig as baseS3Config } from '@aws-sdk/client-s3';
 import {
     debugLogger, has, isEmpty, isNumber
 } from '@terascope/utils';
 import type { S3Client } from './client-types';
 
-export interface S3ClientConfig extends baseConfig {
+export interface S3ClientConfig extends baseS3Config {
```

Comment:
I don't think this inheritance makes sense.  We basically have two things here:

* the Terafoundation S3 connector type
* The AWS S3 client config object

The whole point of the `createS3Client` is to take our connector, map it to a new S3 client config object and us it to instantiate an S3Client object.